### PR TITLE
Create BNFet233.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created stub to be able to refer to it in https://github.com/BetaMasaheft/Works/pull/1185

### CODE REVIEW

File: ParisBNF/et/BNFet233.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                     <altIdentifier>
+                        <idno>Mondon-Vidailhet 46</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+               <msContents>
+                  <summary/>
+                  
+                  <msItem xml:id="ms_i1">
+                     <locus from="1r" to="33v"/>
+                     <title type="complete">Amharic poems</title>
```
Comment (nafisa-valieva) [inline ParisBNF/et/BNFet233.xml:39]: No Lit id?

File: ParisBNF/et/BNFet233.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                     
+                     <msItem xml:id="ms_i2">
+                        <locus from="34r" to="35v"/>
+                        <title type="complete" corresp="LIT3063RepCh343"/>
+                        <textLang mainLang="gez"/>                    
+                  </msItem>
+                  
+                  <msItem xml:id="ms_i3">
+                     <locus from="35v" to="40r"/>
+                     <title type="complete">Amharic treatise on the Trinity</title>
```
Comment (nafisa-valieva) [inline ParisBNF/et/BNFet233.xml:51]: no Lit id?

File: ParisBNF/et/BNFet233.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply ParisBNF/et/BNFet233.xml:39]: As this is just a stub, I don't think it's necessary to go through the process of creating IDs for all works. Would you agree, @eu-genia and @CarstenHoffmannMarburg ?

**Review by eu-genia — APPROVED** [review]

File: ParisBNF/et/BNFet233.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply ParisBNF/et/BNFet233.xml:39]: Yes, I agree. It will be improved in the future.

File: ParisBNF/et/BNFet233.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply ParisBNF/et/BNFet233.xml:51]: I think, it is not neccessary in a stub.

**Pull request merged**